index.html

```diff
@@ -4372,6 +4372,154 @@ <h2>Revision History</h2>
+  &quot;proof&quot;: {&quot;id&quot;: &quot;urn:proof-0&quot;, &quot;verificationMethod&quot;: pubVPE,
+    &quot;proofValue&quot;: signature(privVPE, document)}
+}
+      </code></pre>
+      <p>
+Next, the CFO receives the document, verifies that the VP of engineering signed
+it and signs it based on a review and on the VP of Engineering signing. For this
+we need to set up the <em>proof chain</em> by indicating a dependency on the
+proof in the document just received, i.e.,
+<code>previousProof = &quot;urn:proof-0&quot;</code>. The
+processing algorithm has us &quot;bind&quot; the previous proof to the unsigned
+document. Denote the this proof by <em>proof0</em> and &quot;binding&quot; this
+information to the document by <code>document + proof0</code>:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown
+  &quot;proof&quot;: [
+  {&quot;id&quot;: &quot;urn:proof-0&quot;, &quot;verificationMethod&quot;: pubVPE,
+    &quot;proofValue&quot;: signature(VP_Eng, document)},
+  {&quot;id&quot;: &quot;urn:proof-1&quot;, &quot;verificationMethod&quot;: pubCFO, &quot;previousProof&quot;: &quot;urn:proof-0&quot;,
+    &quot;proofValue&quot;: signature(privCFO, document + proof0)}]
+}
+      </code></pre>
+      <p>
+Now when the CEO validates the received the VC with the above <em>proof
+chain</em> they will check that the CFO signed based on the VP of Engineering
+signing it. First, they will check the proof/signature with
+<code>id = 0</code> against the public key of the VP of Engineering. Note that
+this proof/signature is over the original document. Next, the CEO will check
+the proof/signature <code>id = &quot;urn:proof-1&quot;</code> against the public
+key of the CFO.
+However, to make sure that the CFO signed with the document knowing, and with
+proof, that the VP of engineering signed we verify this signature/proof over
+the combination of the document and <em>proof0</em>. If validation is
+successful the CEO signs producing a proof over the document and
+<em>proof1</em> (the proof with <code>id = &quot;urn:proof-1&quot;</code>).
+The final signed chain looks like:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown
+  &quot;proof&quot;: [
+  {&quot;id&quot;: &quot;urn:proof-0&quot;, &quot;verificationMethod&quot;: pubVPE,
+    &quot;proofValue&quot;: signature(VP_Eng, document)},
+  {&quot;id&quot;: &quot;urn:proof-1&quot;, &quot;verificationMethod&quot;: pubCFO, &quot;previousProof&quot;: &quot;urn:proof-0&quot;,
+    proofValue: signature(privCFO, document + proof0)},
+  {&quot;id&quot;: &quot;urn:proof-2&quot;, &quot;verificationMethod&quot;: pubCEO, &quot;previousProof&quot;: &quot;urn:proof-1&quot;,
+    &quot;proofValue&quot;: signature(privCEO, document + proof1)},
+  ]
+}
+      </code></pre>
+      <p>
+The recipient of this VC then validates it in a similar way as above but with one extra step.
```

Comment:
What's the extra step?
```suggestion
The recipient of this verifiable credential then validates it in a similar way as above, but with one extra step.
```